frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -1201,17 +1146,20 @@ export default ({
       const fromIsJoined = from.isJoined
 
       const initAfterSynced = (toChatRoomId) => {
-        if (toChatRoomId !== this.summary.chatRoomID || this.ephemeral.messagesInitiated) return
-        return this.setInitMessages()
+        if (toChatRoomId !== this.summary.chatRoomID || // If the user has switched to another chatroom during syncing, no need to process the chatroom that has been swithed away.
```

Comment:
Could you please put this comment above if `if` on its own line?

Reply:
done